### PR TITLE
Blur element when country is selected

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
As a user, when typing a country of my choice, I should be able to see the action that allows me to proceed. This happens by default on Chrome for Android, while on Safari iOS the action is covered by the virtual keyboard.

# Solution
Dismiss the mobile keyboard once a country has been selected

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1186-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CountrySelector/index.js [hunk 1]
```diff
@@ -44,16 +44,18 @@ const getCountryOptionTemplate = (country: CountryData) =>
...
   }
 
-  handleCountrySearchConfirm = (selectedCountry: CountryData) => {
+  handleCountrySearchConfirm = async (selectedCountry: CountryData) => {
     if (selectedCountry) {
       this.setState({
         showNoResultsError: false,
       })
-      this.props.actions.setIdDocumentIssuingCountry(selectedCountry)
+      await this.props.actions.setIdDocumentIssuingCountry(selectedCountry)
```
Comment (ethan605) [inline src/components/CountrySelector/index.js:57]: Why you're turning this action call into async? I don't see any network request here? Unless you want to defer this call to next process tick, then I think it should be triggered via `setTimeout(handler, 0)`

**Review by ethan605 — CHANGES_REQUESTED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1186-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CountrySelector/index.js [hunk 2]
```diff
@@ -44,16 +44,18 @@ const getCountryOptionTemplate = (country: CountryData) =>
...
+      this.autocompleteEl.elementReferences[-1].blur()
```
Comment (kopijunkie) [inline src/components/CountrySelector/index.js:58]: What/why does `elementReferences` do & need to be -1?

File: src/components/CountrySelector/index.js (hunk 1 above)
Comment (kopijunkie) [reply src/components/CountrySelector/index.js:57]: Can confirm there's no network requests here for setting the issuing country.

File: src/components/CountrySelector/index.js (hunk 2 above)
Comment (stefaniacardenas) [reply src/components/CountrySelector/index.js:58]: That's the reference of[ the input element ](https://github.com/alphagov/accessible-autocomplete/blob/a148b00c85118361071ec9ecc28a1dc6d905d6c7/src/autocomplete.js#L366). I agree, this is not very readable. I will just get the element by id.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Review by kopijunkie — APPROVED** [review]
LGTM

**Review by ethan605 — APPROVED** [review]
LGTM!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1186-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1186-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**